test/conftest.py

```diff
@@ -252,5 +256,17 @@ def workspace():
 
 @pytest.fixture(autouse=True)
 def conformance_xfail(request):
-    if request.node.originalname in _XFAIL_LIST:
+
+    if any([fnmatch(request.node.name, xfail_pattern) for xfail_pattern in _XFAIL_LIST]):
         request.node.add_marker(pytest.mark.xfail(reason="skipped by suite runner", strict=True))
+
+
+def pytest_generate_tests(metafunc):
+    """
+    Parametrize bundle_verify tests over all sudirectories under assets/bundle-verify
+    """
+    if "bundle_verify_dir" in metafunc.fixturenames:
+        asset_root = Path(__file__).parent / "assets" / "bundle-verify"
+        directories = [d for d in asset_root.iterdir() if d.is_dir()]
+        dir_paths = [str(d) for d in directories]
+        metafunc.parametrize("bundle_verify_dir", dir_paths, ids=[d.name for d in directories])
```

Comment:
Non-blocker: it could be nicer to set the xfail here, like
```suggestion
        dir_paths = [str(d) for d in directories]
        params = [pytest.param(str(d), id=d.name, marks=[pytest.mark.xfail] if str(d).endswith("fail") else [])
        metafunc.parametrize("bundle_verify_dir", params)
```